velox/docs/functions/spark/regexp.rst

```diff
@@ -38,7 +38,9 @@ See https://github.com/google/re2/wiki/Syntax for more information.
 .. spark:function:: regexp_replace(string, pattern, overwrite) -> varchar
 
     Replaces all substrings in ``string`` that match the regular expression ``pattern`` with the string ``overwrite``. If no match is found, the original string is returned as is.
-    There is a limit to the number of unique regexes to be compiled per function call, which is 20.
+    There is a limit to the number of unique regexes to be compiled per function call, which is 20. If this limit is exceeded the function will throw an exception and return null values.
```

Comment:
> If this limit is exceeded the function will throw an exception and return null values.

A function cannot both throw an exception and return null value. I think this should say " throw an exception".